### PR TITLE
Add filter in group membership admin for MFA'd profiles

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: mozillians/groups/admin.py [hunk 1]
```diff
@@ -128,6 +129,29 @@ def queryset(self, request, queryset):
         return queryset
 
 
+class IsMFA(SimpleListFilter):
```
Comment (akatsoulas) [inline mozillians/groups/admin.py]: s/IsMFA/HasMFAEnabled

File: mozillians/groups/admin.py [hunk 2]
```diff
@@ -128,6 +129,29 @@ def queryset(self, request, queryset):
...
+    def lookups(self, request, model_admin):
+        return (('True', 'True'),
+                ('False', 'False'),)
+
+    def queryset(self, request, queryset):
+        if self.value() == 'True':
+            return queryset.filter(userprofile__idp_profiles__type__in=[
+                IdpProfile.PROVIDER_GITHUB, IdpProfile.PROVIDER_LDAP]
+            ).distinct()
+
```
Comment (akatsoulas) [inline mozillians/groups/admin.py]: Let's remove this empty line from the `if elif`.

File: mozillians/groups/admin.py [hunk 3]
```diff
@@ -128,6 +129,29 @@ def queryset(self, request, queryset):
...
+        elif self.value() == 'False':
+            return queryset.exclude(userprofile__idp_profiles__type__in=[
+                IdpProfile.PROVIDER_GITHUB, IdpProfile.PROVIDER_LDAP]
+            ).distinct()
+
```
Comment (akatsoulas) [inline mozillians/groups/admin.py]: same here

**Pull request merged**